### DISCUSSION TITLE
How to apply animated to a custom component which accepts props for rendering

### DISCUSSION THREAD

**Discussion by soumalya004728o** [pmndrs/react-spring#1218] [Support]
I have a custom component in react named `HomeContent` which accepts a number of props. 
As written in official website I did this:
`const animatedHomeContent = animated(HomeContent);`
 Then to render the component I do
`<animatedHomeContent
            id={1}
            avatar="O"
            image="..."
            image_title="Onion"
            title=" Medium Sized Onions"
            subtitle="November 08, 2020"
            desc="Onions from farms of Nasik"
            price="74"
            quantity="1kg"
          />`
But Nothing is getting rendered. I can't understand how to implement it. Please help

**Comment by jvfresco** [comment]
Could you use an example in a sandbox or something ?
